recipes/polyhedral-gravity-model/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is part of the GitHub repo, it shouldn't be needed here.

Reply:
File has been removed